lib/blacklight/solr/search_builder_behavior.rb

```diff
@@ -282,7 +282,7 @@ def facet_value_to_fq_string(facet_field, value)
 
       case
         when (facet_config and facet_config.query)
-          facet_config.query[value][:fq]
+          facet_config.query[value].blank? ? '-*' : facet_config.query[value][:fq]
```

Comment:
Sorry, I wasn't thinking about succinctness, more about clarity and understandability. Maybe something like this?

```
if facet_config.query[value]
  facet_config.query[value][:fq]
else
  # exclude all documents for the bogus facet query key
  '-*:*'
end